code/_onclick/item_attack.dm

```diff
@@ -93,15 +87,11 @@ avoid code duplication. This includes items that may sometimes act as a standard
 		power *= 2
 	if(ishuman(user))
 		var/mob/living/carbon/human/X = user
-		if(X.gloves && istype(X.gloves,/obj/item/clothing/gloves/force))
-			var/obj/item/clothing/gloves/force/G = X.gloves
-			power *= G.amplification
```

Comment:
You didn't delete the variable on the gloves.

Reply:
The proposed changes are: `Force gloves no longer amplify their damage with melee weapons.`

**NOT**: `Force gloves no longer amplify melee damage of any kind.`

Reply:
Why did you keep the variable if you completely deleted the code it's used for...?

Reply:
Because it still exists in other files that I did not touch, on purpose. It will still amplify natural fist or claw damage as proposed.

Reply:
sounds useless but ok